### DISCUSSION TITLE
telebot async with mysql aync question

### DISCUSSION THREAD

**Discussion by sudousercom** [eternnoir/pyTelegramBotAPI#1671] [Q&A]
Hi! Maybe anyone can advise, how can i use mysql async pool with telebot? All modules uses asynс loop, but i tried to use loop with scheduler, and it blocks polling, and bot becomes silent and not react on commands, thanks

**Comment by coder2020official** [comment]
Scheduler is blocking polling? What scheduler are you using, and how?

**Reply by sudousercom** [reply]
scheduler is aioscheduler, but without using loop, i take an example from async timer bot, and it works fine now. Before i check examples, i tried use loop = asyncio.get_event_loop() and this code blocks polling, in async mysql examples i see same construction, and now i am here, asking for better solution

**Reply by coder2020official** [reply]
Never tried to use aioscheduler. I've used apscheduler library and it's AsyncIOScheduler. It worked for me fine

**Reply by coder2020official** [reply]
In order to stop blocking polling, you need to create tasks

**Reply by sudousercom** [reply]
Thanks for answers about scheduler, it work s fine, maybe you can advise how to use mysql async?

**Reply by coder2020official** [reply]
https://pypi.org/project/aiomysql/
